### PR TITLE
EASYOPAC-812 - Implement status module.

### PR DESCRIPTION
**Pull request by NicolaeGrigoriu**

### CODE REVIEW

File: easyddb_status.admin.inc [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+/**
+ * @param array $form
+ *   Form elements in renderable array.
+ * @param array $form_state
+ *   Status of the form elements.
+ *
+ * @return mixed
+ *   Renderable array with all the form elements.
+ */
+// curl https://api.statuspage.io/v1/pages/g6py5c2psjpr.json \-H "Authorization: OAuth fd68825f-da80-4cbb-90bf-cd31277dd593"
```
Comment (bozeak) [inline easyddb_status.admin.inc]: Please remove commented code.

File: css/easyddb_status.css [hunk 2]
```diff
@@ -0,0 +1,132 @@
+/* line 2, ../../../themes/ddbasic/sass/contrib/neat/grid/_grid.scss */
+* {
```
Comment (guddo) [inline css/easyddb_status.css:2]: This resetting really fucks up so many things. We have had bounced in several modules because of this. Why is this needed @NicolaeGrigoriu ?

**Review by guddo — COMMENTED** [review]
Module should be named `easyopac_status` and source code should be changed as well.

**Comment by guddo** [comment]
Also remember to add module in easySuite/ding2.
